cityelf/src/main/java/com/cityelf/service/IRepositoryImpl.java

```diff
@@ -0,0 +1,17 @@
+package com.cityelf.service;
+
+import com.cityelf.model.Notification;
+import com.cityelf.model.User;
+import com.cityelf.repository.IRepository;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by Sergii on 26.04.2017.
+ */
+@Component
+public class IRepositoryImpl implements IRepository {
```

Comment:
It should be Notification Repository with the annotation **@Repository** without any implementations